### DISCUSSION TITLE
Issue with centering

### DISCUSSION THREAD

**Discussion by iMongui** [mriscoc/Ender3V2S1#678] [Q&A]
This afternoon I've tried to upgrade firmware which I think it was a good idea. After I noticed that when I try to print, the first line that it makes on the left side if you see it from the front, then, instead of going to the center and start to print, its printing on the same side than the first left line which is annoying and of course, I need to change. I've tried to restore settings but also fresh restore, same... Changing offsets and so many options that right now I don't have more options to try so I would like ask for a help

Ender 3 v2, ubltouch, board 4.2.2, thanks !

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
